exporter/prometheusexporter/accumulator_test.go

```diff
@@ -216,6 +217,102 @@ func TestAccumulateMetrics(t *testing.T) {
 				dp.SetTimestamp(pdata.NewTimestampFromTime(ts))
 			},
 		},
+		{
+			name: "Summary",
+			metric: func(ts time.Time, v float64, metrics pdata.MetricSlice) {
+				metric := metrics.AppendEmpty()
+				metric.SetName("test_metric")
+				metric.SetDataType(pdata.MetricDataTypeSummary)
+				metric.SetDescription("test description")
+				dp := metric.Summary().DataPoints().AppendEmpty()
+				dp.SetCount(10)
+				dp.SetSum(0.012)
+				dp.SetCount(10)
+				dp.Attributes().InsertString("label_1", "1")
+				dp.Attributes().InsertString("label_2", "2")
+				dp.SetTimestamp(pdata.NewTimestampFromTime(ts))
+				fillQuantileValue := func(pN, value float64, dest pdata.ValueAtQuantile) {
+					dest.SetQuantile(pN)
+					dest.SetValue(value)
+				}
+				fillQuantileValue(0.50, 190, dp.QuantileValues().AppendEmpty())
+				fillQuantileValue(0.99, 817, dp.QuantileValues().AppendEmpty())
+			},
+		},
+		{
+			name: "StalenessMarkerGauge",
+			metric: func(ts time.Time, v float64, metrics pdata.MetricSlice) {
+				metric := metrics.AppendEmpty()
+				metric.SetName("test_metric")
+				metric.SetDataType(pdata.MetricDataTypeGauge)
+				metric.SetDescription("test description")
+				dp := metric.Gauge().DataPoints().AppendEmpty()
+				dp.SetDoubleVal(v)
+				dp.Attributes().InsertString("label_1", "1")
+				dp.Attributes().InsertString("label_2", "2")
+				dp.SetTimestamp(pdata.NewTimestampFromTime(ts))
+				dp.SetFlags(1)
```

Comment:
```suggestion
				dp.SetFlags(pdata.MetricDataPointFlagNoRecordedValue)
```

Use the defined constant so that future readers have a better chance of understanding what this test is doing.

Reply:
Thank you for guidance. I once tried it but the types are different so in this case I need to cast it like below. Should I change like this?
```go
dp.SetFlags(pdata.MetricDataPointFlags(pdata.MetricDataPointFlagNoRecordedValue))```